pages/src.html

```diff
@@ -125,7 +125,23 @@
     ## Organization
 
     - **Chair:** Saugata Ghose (Univ. of Illinois Urbana-Champaign)
-    - **Selection Committee:** *to be determined*
+    - **Selection Committee:**
```

Comment:
Do you want to repeat this on the org page? It could be factored out into an included file.

Reply:
That's up to you. I don't think it's been shown there in years past, but if it wouldn't make the org page too messy, then I'm happy to make it happen

Reply:
If you feel like their contribution deserves a mention on the org page, I'd be happy to have it.